tests/test_validators.py

```diff
@@ -1351,13 +1351,16 @@ def check_a(cls, v: Any):
 
     with pytest.raises(ValidationError) as exc_info:
         Model(a='snap')
-    injected_by_pytest = "assert 'snap' == 'a'\n  - a\n  + snap"
+    if pytest.version_tuple > (8,):
```

Comment:
IMO this test shouldn't have worked this way in the first place and now this is making it more complicated. Please change `assert v == 'a', 'invalid a'` to `if v != a: raise AssertionError('invalid a')` and then pytest shouldn't inject anything and the test can be simplified.

Reply:
Thanks. Done that.